### PR TITLE
feat: Add additional port for `metrics-server` to recommended rules

### PR DESCRIPTION
**Pull request by bryantbiggs**
## Description
- Add additional port for `metrics-server` to recommended rules

## Motivation and Context
- `metrics-server` is a common utility in EKS clusters

## Breaking Changes
- No

## How Has This Been Tested?
- [ ] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [ ] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [x] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

**Review by antonbabenko — APPROVED** [review]

**Comment by antonbabenko** [comment]
This PR is included in version 19.3.0 :tada:

File: node_groups.tf [hunk 1]
```diff
@@ -115,6 +115,16 @@ locals {
       type        = "ingress"
       self        = true
     }
+    # metrics-server
+    ingress_cluster_8443_webhook = {
```
Comment (pavlospt) [inline node_groups.tf:119]: @bryantbiggs 8443 is typo in the SG name!

Comment (bryantbiggs) [reply node_groups.tf:119]: gosh dangit - thanks!

Comment (bryantbiggs) [reply node_groups.tf:119]: #2354 should resolve - thanks for catching

**Pull request merged**